openlibrary/templates/books/add.html

```diff
@@ -55,6 +55,8 @@ <h1>$_("Add a book to Open Library")</h1>
         <div class="formElement">
             <div class="label"><label for="id_value">$:_("And optionally, an ID number &#151; like an ISBN &#151; would be helpful...")</label></div>
                 <label for="id_name" class="hidden">$_("ID Type")</label>
+                <div id="id-errors" class="note" style="display: none"></div>
+                <div id="confirm-add" class="note" style="display: none"></div>
```

Comment:
```suggestion
                <div id="id-errors" class="note hidden"></div>
                <div id="confirm-add" class="note hidden">$_('ISBN may be invalid. Click "Add" again to submit.')</div>
```
We try to avoid adding new inline styling to our templates.  We have a `.hidden` class that can be used here to set display to `none`.
The message in the `#confirm-add` div will never change, so it can be added here as a localized string (more details about localization can be found in this [README](https://github.com/internetarchive/openlibrary/tree/master/openlibrary/i18n)).